WordPress/Classes/ViewRelated/Post/PostListViewController.swift

```diff
@@ -585,114 +552,115 @@ class PostListViewController: AbstractPostListViewController, UIViewControllerRe
+            return NoResultsText.noDraftsTitle
         case .scheduled:
-            return NSLocalizedString("Would you like to create one?", comment: "Displayed when the user views scheduled posts in the posts list and there are no posts")
+            return NoResultsText.noScheduledTitle
         case .trashed:
-            return NSLocalizedString("Everything you write is solid gold.", comment: "Displayed when the user views trashed posts in the posts list and there are no posts")
-        default:
-            return NSLocalizedString("Would you like to publish your first post?", comment: "Displayed when the user views published posts in the posts list and there are no posts")
-        }
-    }
-
-    // MARK: - UISearchControllerDelegate
-
-    override func willPresentSearchController(_ searchController: UISearchController) {
-        super.willPresentSearchController(searchController)
-
-        self.filterTabBar.alpha = WPAlphaZero
-        filterTabBarBottomConstraint.isActive = false
-        tableViewTopConstraint.isActive = true
-    }
-
-    func didPresentSearchController(_ searchController: UISearchController) {
-        if #available(iOS 11.0, *) {
-            updateTableHeaderSize()
-
-            tableView.scrollIndicatorInsets.top = searchWrapperView.bounds.height
-            tableView.contentInset.top = 0
+            return NoResultsText.noTrashedTitle
+        case .published:
+            return NoResultsText.noPublishedTitle
         }
     }
 
-    func didDismissSearchController(_ searchController: UISearchController) {
-        updateTableHeaderSize()
-
-        tableViewTopConstraint.isActive = false
-        filterTabBarBottomConstraint.isActive = true
-
-        UIView.animate(withDuration: Animations.searchDismissDuration) {
-            self.filterTabBar.alpha = WPAlphaFull
-        }
+    struct NoResultsText {
+        static let buttonTitle = NSLocalizedString("Create a Post", comment: "Button title, encourages users to create their first post on their blog.")
+        static let fetchingTitle = NSLocalizedString("Fetching posts...", comment: "A brief prompt shown when the reader is empty, letting the user know the app is currently fetching new posts.")
+        static let noMatchesTitle = NSLocalizedString("No posts matching your search", comment: "Displayed when the user is searching the posts list and there are no matching posts")
+        static let noDraftsTitle = NSLocalizedString("You don't have any draft posts", comment: "Displayed when the user views drafts in the posts list and there are no posts")
+        static let noScheduledTitle = NSLocalizedString("You don't have any scheduled posts", comment: "Displayed when the user views scheduled posts in the posts list and there are no posts")
+        static let noTrashedTitle = NSLocalizedString("You don't have any binned posts", comment: "Displayed when the user views trashed in the posts list and there are no posts")
```

Comment:
Did editorial write this?

Reply:
Well now, I honestly don't know - I got it from the Zeplin project. 

@SylvesterWilmott - what do you think about using "trashed" here instead of "binned"?

Reply:
It looks like Android went with "trashed", so I will follow suit. If there's any dissension I can change it later.